src/components/preview/preview.stories.mdx

```diff
@@ -1,7 +1,9 @@
+import { useState } from "react";
 import { Meta, Story, Canvas } from "@storybook/addon-docs";
 import { ArgsTable } from "@storybook/components";
 import PreviewComponent from "./preview.component";
```

Comment:
```suggestion
import Preview from "./preview.component";
```
I would suggest we change the import name so it's not confusing for consumers as they read our docs